DataCert/dataCertPlots.py

```diff
@@ -361,54 +361,61 @@ def GetYAverage(hist,zeroSupress=True):
-        PCLumiOBest.Draw("hist")
-        PCLumiOHF.Draw("histsame")
-        PCLumiOBCM1F.Draw("histsame")
-        bestOHFLumi.Draw("histsame")
-        hfOverBcm1f.Draw("histsame")
-        pltOverHF.Draw("histsame")
+        BestOPCLumi=histbest[run].Clone("best_o_PCLumi")
+        BestOPCLumi.Divide(histPCLumiB3p8[run])
+        BestOPCLumi.SetMaximum(2.0)
+        BestOPCLumi.SetTitle(";Luminosity Section  ;Ratios")
+        BestOPCLumi.SetLineColor(802)
+        mean1,meanError1=GetYAverage(BestOPCLumi,True)
+##hf
+        HFOPCLumi=histHFLumi[run].Clone("HF_o_PCLumi")
+        HFOPCLumi.Divide(histPCLumiB3p8[run])
+        HFOPCLumi.SetMaximum(2.0)
+        HFOPCLumi.SetTitle(";Luminosity Section  ;HF/PCLumi")
+        HFOPCLumi.SetLineColor(800)
+        mean2,meanError2=GetYAverage(HFOPCLumi,True)
+##BCM1F
+        BCM1FOPCLumi=histBCMFLumi[run].Clone("BCM1F_o_PCLumi")
+        BCM1FOPCLumi.Divide(histPCLumiB3p8[run])
+        BCM1FOPCLumi.SetMaximum(2.0)
+        BCM1FOPCLumi.SetTitle(";Luminosity Section  ;BCM1F/PCLumi")
+        BCM1FOPCLumi.SetLineColor(617)
+        mean3,meanError3=GetYAverage(BCM1FOPCLumi,True)
+##PLT
+        PLTOPCLumi=histPLTLumi[run].Clone("PLT_o_PCLumi")
+        PLTOPCLumi.Divide(histPCLumiB3p8[run])
+        PLTOPCLumi.SetMaximum(2.0)
+        PLTOPCLumi.SetTitle(";Luminosity Section  ;PLT/PCLumi")
+        mean4,meanError4=GetYAverage(PLTOPCLumi,True)
+##HF
+        hfOverPLT=histHFLumi[run].Clone("HF_o_PLT")
+        hfOverPLT.Divide(histPLTLumi[run])
+        hfOverPLT.SetMaximum(2.0)
+        hfOverPLT.SetTitle(";Luminosity Section  ;HF/PLTLumi")
+        mean5,meanError5=GetYAverage(hfOverPLT,True)
+##BCM1F
+        bcmfOverPLT=histBCMFLumi[run].Clone("BCM1F_o_PLT")
+        bcmfOverPLT.Divide(histPLTLumi[run])
+        bcmfOverPLT.SetMaximum(2.0)
+        bcmfOverPLT.SetTitle(";Luminosity Section  ;BCM1F/PLTLumi")
+        mean6,meanError6=GetYAverage(bcmfOverPLT,True)
+
+
+ 	leg2=ROOT.TLegend(0.1,0.7,0.5,0.9)
+        leg2.AddEntry(BestOPCLumi, "Best/PCLumi = 	"+"{:10.4f}".format(mean1)+" #pm "+"{:10.4f}".format(meanError1),"l")
+        leg2.AddEntry(HFOPCLumi,   "HF/PCLumi = 	"+"{:10.4f}".format(mean2)+" #pm "+"{:10.4f}".format(meanError2),"l")
+        leg2.AddEntry(BCM1FOPCLumi,"BCM1F/PCLumi = 	"+"{:10.4f}".format(mean3)+" #pm "+"{:10.4f}".format(meanError3),"l")
+        leg2.AddEntry(PLTOPCLumi,  "PLT/PCLumi = 	"+"{:10.4f}".format(mean4)+" #pm "+"{:10.4f}".format(meanError4),"l")
+        leg2.AddEntry(hfOverPLT,   "HF/PLT = 		"+"{:10.4f}".format(mean5)+" #pm "+"{:10.4f}".format(meanError5),"l")
+        leg2.AddEntry(bcmfOverPLT, "BCM1F/PLT = 	"+"{:10.4f}".format(mean6)+" #pm "+"{:10.4f}".format(meanError6),"l")
+        BestOPCLumi.Draw("hist")
+        HFOPCLumi.Draw("histsame")
+        BCM1FOPCLumi.Draw("histsame")
+        PLTOPCLumi.Draw("histsame")
+        hfOverPLT.Draw("histsame")
+        bcmfOverPLT.Draw("histsame")
         leg2.Draw("sames")
         padlumis.Update()
 
```

Comment:
There is definitely a problem with alignment here.  Can you make sure there are no tabs?